src/globals/internal/FloatingMenu.js

```diff
@@ -225,16 +231,18 @@ class FloatingMenu extends React.Component {
       oldRefPosition.bottom !== refPosition.bottom ||
       oldRefPosition.left !== refPosition.left ||
       hasChangeInOffset(oldMenuOffset, menuOffset) ||
-      oldMenuDirection !== menuDirection
+      oldMenuDirection !== menuDirection ||
+      !this.state.menuRendered
     ) {
       const menuSize = menuBody.getBoundingClientRect();
       const offset =
         typeof menuOffset !== 'function' ? menuOffset : menuOffset(menuBody, menuDirection);
       // Skips if either in the following condition:
       // a) Menu body has `display:none`
       // b) `menuOffset` as a callback returns `undefined` (The callback saw that it couldn't calculate the value)
-      if ((menuSize.width > 0 && menuSize.height > 0) || !offset) {
+      if (menuSize.width > 0 && menuSize.height > 0 && offset) {
```

Comment:
i didn't know whether to change the comment or the functionality but now they at least align (and I agree with the comment's sentiment so I changed the code to match)